src/StataOracle.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.10;
+
+import {IPool} from 'aave-v3-core/contracts/interfaces/IPool.sol';
+import {IPoolAddressesProvider} from 'aave-v3-core/contracts/interfaces/IPoolAddressesProvider.sol';
+import {IAaveOracle} from 'aave-v3-core/contracts/interfaces/IAaveOracle.sol';
+import {IERC20Metadata} from 'solidity-utils/contracts/oz-common/interfaces/IERC20Metadata.sol';
+import {IStaticATokenLM} from './interfaces/IStaticATokenLM.sol';
+import {IStataOracle} from './interfaces/IStataOracle.sol';
+import {IERC4626} from './interfaces/IERC4626.sol';
+
+/**
+ * @title StataOracle
+ * @author BGD Labs
+ * @notice Contract to get asset prices of stata tokens
+ */
+contract StataOracle is IStataOracle {
+  IPoolAddressesProvider public immutable POOL_ADDRESSES_PROVIDER;
```

Comment:
maybe not needed, because not used outside of constructor, it's not even exposed on the interface